Pipeline/D - Model sensitivity analyses and checks/D03 - Reviewer sensitivity checks.R

```diff
@@ -0,0 +1,327 @@
+
+myforms = c(cXt2intrXm, cXt2intrXmDM, cXt2intrXmSM) 
+
+mycollabs = c(
+  "main specification", # Main Spec
+  "+ diag. method (full set)", # Main Spec with dominant method
+  "+ diag. method (small set)" # Main Spec with simplified method
+)
+
+modellist = list()
+i=0
+for (m in myforms) {
+  i=i+1
+  modellist[[i]] = felm(data = complete_dm, formula = m)
+}
+
+mynote = "Column specifications: (1) country-specific quad. trends, intervention year FE, GBOD region-by-month FE; (2) same as (1), but with additional controls for diagnostic method: Microscopy, Microcscopy/PCR Confirmed, PCR, RDT, RDT/PCR Confirmed, and RDT/SLIDE Confirmed; (3) same as (2) but using simplified diagnostic method control set: Microscopy, PCR, RDT."
+dir.create(file.path(resdir, "Tables", "sensitivity"), showWarnings = FALSE)
+stargazer(
+  modellist,
+  title = "Sensitivity to controlling for diagnostic method", 
+  align = TRUE, 
+  column.labels = mycollabs,
+  keep = c("temp", "flood", "drought", "METHOD"),
+  out = file.path(resdir, "Tables", "sensitivity","DiagMethod_sensitivity.tex"),  
+  omit.stat = c("f", "ser"), 
+  out.header = FALSE,
+  type = "latex", 
+  float = F,
+  notes.append = TRUE, 
+  digits = 2,
+  notes.align = "l", 
+  notes = paste0("\\parbox[t]{\\textwidth}{", mynote, "}")
+)
+
+####### Plot temperature responses #######
+
+plotXtemp = cbind(seq(Tmin,Tmax), seq(Tmin,Tmax)^2)
+
+figList = list()
+for(m in 1:length(modellist)) {
+  coefs = summary(modellist[[m]])$coefficients[1:2]
+  myrefT = max(round(-1*coefs[1]/(2*coefs[2]), digits = 0), 10)
+  figList[[m]] = plotPolynomialResponse(
+    modellist[[m]], "temp", plotXtemp,
+    polyOrder = 2, cluster = T, xRef = myrefT,
+    xLab = expression(paste("Mean temperature (",degree,"C)")), 
+    yLab = "Prevalence (%)", title = mycollabs[m], yLim=c(-30,10), showYTitle = T) +
+    theme(plot.title = element_text(size = 10))
+}
+
+p = plot_grid(figList[[1]], figList[[2]], figList[[3]], nrow=1)
+p
+
+fe_fig_dir <- file.path(resdir, "Figures", "Diagnostics","Fixed_effects")
+dir.create(fe_fig_dir, showWarnings = FALSE)
+ggsave(
+  filename = "diagnostic_method_sensitivity.pdf",
+  path = fe_fig_dir, 
+  plot = p, 
+  width = 9, 
+  height = 3
+)
+
+########################################################################
+# R2: Correlated errors
+########################################################################
+
+# Are residuals correlated within countries? 
+resCntry = felm(res ~ monthyr + I(country), data=complete)
+pvals = summary(resCntry)$coefficients[,"Pr(>|t|)"]
+ph = ggplot() + 
+  geom_histogram(aes(x=pvals), color= "seagreen", fill = "seagreen") + 
+  xlab("p-values by country") +
+  theme_classic()
+ph
+
+ggsave(file.path(resdir, "Figures", "Diagnostics", "Reviewer_comments", "res_corr_countries.pdf"), plot = g, width = 7, height = 7)
```

Comment:
It looks like this is meant to have `plot = ph` rather than `plot = g`

Reply:
good catch! fixed.